l10n/configs/pontoon.toml

```diff
@@ -115,6 +115,55 @@ locales = [
     "zu",
 ]
 
+[[paths]]
+    reference = "en/*.ftl"
+    l10n = "{locale}/*.ftl"
+[[paths]]
+    reference = "en/banners/**/*.ftl"
+    l10n = "{locale}/banners/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/*.ftl"
+    l10n = "{locale}/firefox/*.ftl"
+[[paths]]
+    reference = "en/firefox/browsers/**/*.ftl"
+    l10n = "{locale}/firefox/browsers/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/features/**/*.ftl"
+    l10n = "{locale}/firefox/features/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/new/**/*.ftl"
+    l10n = "{locale}/firefox/new/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/nightly/**/*.ftl"
+    l10n = "{locale}/firefox/nightly/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/privacy/book.ftl"
+    l10n = "{locale}/firefox/privacy/book.ftl"
+    locales = [
+        "de",
+        "fr"
+    ]
+[[paths]]
+    reference = "en/firefox/products/**/*.ftl"
+    l10n = "{locale}/firefox/products/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/set-as-default/**/*.ftl"
+    l10n = "{locale}/firefox/set-as-default/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/**/*.ftl"
+    l10n = "{locale}/firefox/welcome/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/whatsnew/**/*.ftl"
+    l10n = "{locale}/firefox/whatsnew/**/*.ftl"
+[[paths]]
+    reference = "en/mozorg/about/**/*.ftl"
```

Comment:
Doesn't this one include the next two, or is it really only those within subfolders of `about`?

Reply:
Oh, you're right - shows how brittle this can be :/